### ISSUE TITLE
VS Code consumes 99% CPU

### ISSUE THREAD

**Issue by rebornix** [Microsoft/vscode#13896]
- VSCode Version: 1.6.1
- OS Version: Windows

This is copied from https://www.zhihu.com/question/51673506 and I raised it here as I'm not sure whether there is a duplicate of it already. 

The user only installs TFS extension. I'll put more info here once I reach out to him.

![image](https://cloud.githubusercontent.com/assets/876920/19447085/c2c246d0-9450-11e6-8fc5-b356b84ebf7d.png)

**Comment by rebornix** [comment]
The author mentioned he just launched VS Code and it consumes most of the CPU.

**Comment by roblourens** [comment]
Can he try with the extension disabled? Otherwise I'm not sure how to direct it.

**Comment by rebornix** [comment]
@roblourens I've asked him and will put more info here later.

**Comment by weinand** [comment]
@rebornix I'm assigning this to you because said that you want to add more info. Reassign after you've done that (or close).

**Comment by OllyHodgson** [comment]
I'm seeing what appears to be the same symptoms on Windows 2012 R2 x64 (in a Hyper-V VM). I've tried 1.72 and today's 1.8.0 Insiders build.

I'm using Windows Server 2012 R2 x64. When I start VS Code on the host machine, it appears to work fine. But when I start it in a Hyper-V virtual machine (generation 2 VM - also running Server 2012 R2 x64), CPU usage on one of VS Code's processes quickly climbs to 99% and stays there.

The 1.8.0 Insiders build doesn't do it straight away - but leave it running for a while and it happens eventually.

I'm behind a corporate proxy (uses a PAC file). No anti-virus on the host machine or VM. I've tried the `--disable-gpu` argument, which made no difference. My user settings look like this:

```
{
    "git.enabled": false,
    "http.proxy": "http://url-of-our-proxy:8081",
    "https.proxy": "http://url-of-our-proxy:8081",
    "http.proxyStrictSSL": false,
    "files.exclude": {
        "**/largeFolder": true
    }
}
```

What more info can I provide to help debug this?
